pkg/sdk/identifier_helpers.go

```diff
@@ -165,6 +165,41 @@ func (i SchemaIdentifier) FullyQualifiedName() string {
 	return fmt.Sprintf(`"%v"."%v"`, i.databaseName, i.schemaName)
 }
 
+type DatabaseObjectIdentifier struct {
```

Comment:
good point

Reply:
Well, Schema resides in a database, so in general the answer is yes. I did not want to do a big rename and at the same time I did not want to use SchemaIdentifier (which name is misleading), but I feel that these two types could be merged into one. Maybe in a separate change (because it will be a sh*tload of files changed).

Or maybe you have another suggestion for these two types?

Reply:
Looks good to me